plugins/outputs/graylog/graylog.go

```diff
@@ -319,7 +362,7 @@ func (g *Graylog) serialize(metric telegraf.Metric) ([]string, error) {
 	m["version"] = "1.1"
 	m["timestamp"] = float64(metric.Time().UnixNano()) / 1_000_000_000
 	m["short_message"] = "telegraf"
-	m["name"] = metric.Name()
+	m["_name"] = metric.Name()
```

Comment:
Was this working before? If so you change the output and thus will break existing setups won't you?